### DISCUSSION TITLE
May I ask if this is only for Chinese speakers?

### DISCUSSION THREAD

**Discussion by younes-io** [GaiZhenbiao/ChuanhuChatGPT#1019] [Q&A]
I tried the online demo but everything is in Chinese and I had to click every button to find this popup which suggested languages:

![image](https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/3153107/6fafb5d1-3478-4bd3-84dd-08a244aed037)

When I selected English, nothing changed, the UI is till in Chinese. So, I just want to know if this is only for Chinese speakers or if it's an international app.

Thank you,

**Comment by GaiZhenbiao** [comment]
No, we support many languages, and English is a priority. To use English, you have to duplicate the space. When you are duplicating the space, Hugging face will prompt you to fill in some secrets, you can set the `LANGUAGE` space secret to `en_US`, or do nothing to use the default value (which is `en_US`).

**Reply by younes-io** [reply]
How to duplicate the space ? xD I don't speak Chinese to know where is the "space"... Is this document ? Maybe I missed it in the docs..

**Reply by GaiZhenbiao** [reply]
On the top banner of the hugging face page, their is a ... button. Under that menu, there is a "Duplicate this Space" option.

**Reply by younes-io** [reply]
Oh, the HuggingFace space.. sorry, I thought the app had a concept called "Space"..
Thank you so much !

**Comment by younes-io** [comment]
I think it would be great if there was a popup at the very beginning that requests the user to pick their language; that way, the user is forced to pick the language before continuing.. i don't know what you think about it..
